packages/ses/test/hermes-smoke.js

```diff
@@ -0,0 +1,53 @@
+// Hermes doesn't support native I/O,
```

Comment:
The filename may need a prefix `_` to appease our linter, since all files are now either `*.test.js` or explicitly marked as non-ava fixtures with the `_` prefix.